src/main/java/org/craftercms/studio/api/v2/event/content/MoveContentEvent.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2007-2022 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.craftercms.studio.api.v2.event.content;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.craftercms.studio.api.v2.event.BroadcastEvent;
+import org.craftercms.studio.api.v2.event.SiteAwareEvent;
+import org.springframework.security.core.Authentication;
+
+/**
+ * Event triggered when there is a content move change
+ *
+ * @author Phil Nguyen
+ * @since 4.0.1
+ */
+
+public class MoveContentEvent extends SiteAwareEvent implements BroadcastEvent {
```

Comment:
@phuongnq this class should inherit from `ContentEvent` the idea is to eventually have specific classes for all events and turn `ContentEvent` into an abstract class

Reply:
Fixed to inherit from `ContentEvent`. Thank you!